src/Microsoft.AspNetCore.Authentication/RemoteAuthenticationResult.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Security.Claims;
+
+namespace Microsoft.AspNetCore.Authentication
+{
+    /// <summary>
+    /// Contains the result of an Authenticate call
+    /// </summary>
+    public class RemoteAuthenticationResult : AuthenticateResult
```

Comment:
No, it's really specific to the remote authentication handler. It doesn't make any sense to use it for `IAuthenticationRequestHandler.HandleRequestAsync()` as this method is not meant to return authentication tickets (or failed results).